Dockerfile

```diff
@@ -1,21 +1,24 @@
 FROM alpine:3.19 AS base_gcc
 
-RUN apk add --update alpine-sdk git wget
+RUN apk add --update alpine-sdk git wget sdl2-dev sdl2_mixer-dev
```

Comment:
The primary reason for excluding SDL2 packages from the current Alpine Linux image is to maintain a smaller image size, which enables quicker startup and emulation through [blink](https://github.com/jart/blink) or similar emulators. Including SDL2 in the Alpine Linux image would create challenges when attempting to run graphics with the emulator.